dockerfiles/enclave.debug.Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.17-alpine
+
+# Debug image which installs and runs delve
+#
+# on the container:
+#   /data                    contains working files for the enclave
+#   /home/obscuro/go-obscuro contains the src
+#
+# Note: ego uses a virtual file system mount to map data directory to /data inside the enclave,
+#   for this non-ego build I'm using /data as the data dir to preserve /data folder in paths inside enclave
```

Comment:
Maybe clarify further up that this dockerfile runs the enclave outside of EGo?

Reply:
yep 👍